main.js

```diff
@@ -0,0 +1,81 @@
+var mealsJSON = {
+  'meals': {
+    'breakfast':[
+      {
+        'name': 'Breakfast Burrito',
+        'ingredients': ['tortilla', 'eggs', 'potatoes', 'cheese', 'onion']
```

Comment:
if you make the ingredients an object it would be easier to change the quantities